### ISSUE TITLE
[feature request] RENAME DATABASE

### ISSUE THREAD

**Issue by beckettsean** [influxdb/influxdb#4154]
From https://github.com/influxdb/influxdb/issues/828#issuecomment-83224779

e.g. `RENAME DATABASE foo TO bar`

Would also need to update all relevant CQs.

**Comment by linearb** [comment]
From discussion on https://github.com/influxdb/influxdb/pull/4291, will use this syntax:
ALTER DATABASE name RENAME TO new_name

**Comment by jackzampolin** [comment]
Looks like this is now merged.  Closing.

**Comment by beckettsean** [comment]
Jack, I believe the merge was reverted. @otoolep what's the status of `ALTER DATABASE`?

**Comment by corylanou** [comment]
I believe we decided that we may add an `alias` method, but likely not an `alter` or `rename`.

**Comment by otoolep** [comment]
The feature was reverted, a person in the community may submit a new patch at a later date, but no work planned by us right now.

**Comment by jackzampolin** [comment]
Sorry about that!  Thanks for the catch @beckettsean

**Comment by hpbieker** [comment]
Is there a workaround for this? Such as:
* Stop Influx DB
* Rename database and wal folder to reflect the new name of the database
* Start  Influx DB

I guess it will not work, as it looks like meta/meta.db contains some references to my database?

**Comment by stduolc** [comment]
@hpbieker do you find some command for rename a database?

**Comment by mitar** [comment]
Still interested in this.

**Comment by mitar** [comment]
Still interested in this.

**Comment by svanscho** [comment]
Yeah, would love and need this feature for db migrations we do quite often.

**Comment by mitar** [comment]
Still interested in this.

**Comment by stephenrs** [comment]
I'm also interested in this

**Comment by sultanksh** [comment]
I am also interested in this feature. Rename comes in handy for migration

**Comment by argyris-gardelakos** [comment]
I am also interested in this

**Comment by nvolker** [comment]
This would be great!

**Comment by lsitzmann** [comment]
Another vote for this feature from me :-)
